src/slugline.scss

```diff
@@ -1,26 +1,28 @@
+@import "slugline_vars";
+
+@import "~bootstrap/scss/bootstrap.scss";
+
 @font-face {
-  font-family: "DIN Next";
-  src: url("./fonts/din_next.woff") format("woff");
+  font-family: "Body Font";
+  src: url("https://fonts.googleapis.com/css2?family=Merriweather&display=swap");
 }
 
 @font-face {
-  font-family: "Joanna Nova";
-  src: url("./fonts/joanna_nova.woff") format("woff");
+  font-family: "Headings Font";
+  src: url('https://fonts.googleapis.com/css2?family=Overpass&display=swap');
 }
```

Comment:
I realize this is confusing, but these font face rules should really be put in a separate file for configuration. This second file would then be imported only once, in `slugline.scss`, while the variables defining the font names stay in `_slugline_vars` and are thus visible to the entire project.